webrender/src/device/gfx/device.rs

```diff
@@ -2406,12 +2432,6 @@ impl<B: hal::Backend> Device<B> {
         self.update_instances(&[data]);
 
         self.ensure_blit_program(view_kind);
-        let ref desc_set_per_group = self
-            .bound_per_group_descriptors[descriptor_group as usize]
-            .as_ref()
-            .unwrap()
-            .raw();
-
         let program = self.blit_programs.get_mut(&view_kind).unwrap();
         unsafe {
             self.command_buffer.bind_graphics_pipeline(
```

Comment:
are we ensuring somewhere that these direct bindings aren't going out of sync with the state we know of? i.e. the code that follows would assume the bound pipeline and the descriptor sets are unchanged?

Reply:
Added state tracking for this here: https://github.com/szeged/webrender/pull/350/commits/b0be810ff981417d41a2600b6a4cd74ae5f9a2af#diff-cdf03c96bd34cb98c2d09b0be7658232R2443 . and here https://github.com/szeged/webrender/pull/350/commits/b0be810ff981417d41a2600b6a4cd74ae5f9a2af#diff-cdf03c96bd34cb98c2d09b0be7658232R2463-R2471